common/src/address.rs

```diff
@@ -0,0 +1,19 @@
+use std::convert::{Into, TryFrom};
+//use std::net::SocketAddr;
+use std::path::PathBuf;
+
+use tokio_vsock::VsockAddr;
+
+use crate::pb;
+
+#[derive(Clone, Debug, PartialEq)]
+pub enum EndpointAddress {
+    Tcp {
+        // IP + port (FIXME: should be SocketAddres)
```

Comment:
Using `String` has the benefit of being able to use names instead of IP-addresses.